servers/apps/librechat-echo/README.md

```diff
@@ -0,0 +1,13 @@
+## LibreChat
+
+LibreChat is a self-hosted solution designed for organization members to browse essential information using AI providers. The service is powered by the OpenAI API, with API usage facilitated through periodically purchased credits.
+
```

Comment:
could you please also mention what is the server name on linode ? With the specifications of the server ? CPU, Ram etc ? Just to keep things transparent and also new people would be able to understand that faster.

Reply:
Added.